src/System.Management.Automation/utils/PSTelemetryWrapper.cs

```diff
@@ -18,7 +18,7 @@ internal static class TelemetryWrapper
         /// <summary>
         /// Performing EventSource initialization in the Static Constructor since this is thread safe.
         /// Static constructors are guaranteed to be run only once per application domain, before any instances of a class are created or any static members are accessed.
-        /// https://msdn.microsoft.com/en-us/library/aa645612(VS.71).aspx
+        /// https://msdn.microsoft.com/library/aa645612(VS.71).aspx
```

Comment:
That link points to retired content.   This one is probably more appropriate:

[Static Constructors (C# Programming Guide)](https://docs.microsoft.com//dotnet/csharp/programming-guide/classes-and-structs/static-constructors)

Reply:
Good point. I was wondering if it was worth updating a link about Visual Studio 2003 but your link makes more sense (there were a few places with a similar link)